taglib/toolkit/tstring.cpp

```diff
@@ -38,7 +38,7 @@ namespace
   using namespace TagLib;
 
   // Returns the native format of std::wstring.
-  String::Type wcharByteOrder()
+  constexpr String::Type wcharByteOrder()
   {
     if(Utils::systemByteOrder() == Utils::LittleEndian)
```

Comment:
can't use if constexpr?

Reply:
I will use the ternary operator.

Reply:
I tried this. Looks like it can't:

```
error: accessing ‘TagLib::Utils::{anonymous}::systemByteOrder()::IntCharUnion::c’ member instead of initialized ‘TagLib::Utils::{anonymous}::systemByteOrder()::IntCharUnion::i’ member in constant expression
```

too bad std::endian is not available.